.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+# This is a basic workflow for shellcheck 
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  shellcheck:
+    name: Shellcheck
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run ShellCheck
+      uses: ludeeus/action-shellcheck@master
+
+# for more options https://github.com/marketplace/actions/shellcheck
+  bats:
+    name: Bats testing
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Bats
+        run: sudo apt-get install bats
+
+      - uses: actions/checkout@v2
+      - name: Test Bats 
+        run: bats test_almalinux-deploy.bats
```

Comment:
Can you please add a new line to disable the warning?

Reply:
Sorry, I did not get this one. There is no warning in the CI but rather an error , since I use ubuntu-latest bats testing will fail.

Reply:
Just a simple newline at the end ?

Reply:
Yes, exactly. If you check the https://github.com/AlmaLinux/almalinux-deploy/pull/37/files page you will see a red circle on the 35th line saying that a new line is missing.

Reply:
perfect, done.